Source: Upstream.
Upstream: Yes. Will be in 4.8.4.
Reason: Fix compile with Qt 4.7.

From be5a4a402eb10ed5c30721118fbe9dd151db8825 Mon Sep 17 00:00:00 2001
From: Montel Laurent <montel@kde.org>
Date: Fri, 4 May 2012 11:40:22 +0200
Subject: [PATCH] Fix compile with qt4.7 (cherry picked from commit
 f23ba2a2439dbc3baf7d51751ff82bf215e40f7a)

---
 messageviewer/viewer_p.cpp |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/messageviewer/viewer_p.cpp b/messageviewer/viewer_p.cpp
index 1002b5b..a0ab46b 100644
--- a/messageviewer/viewer_p.cpp
+++ b/messageviewer/viewer_p.cpp
@@ -1930,8 +1930,10 @@ void ViewerPrivate::slotToggleHtmlMode()
 
 void ViewerPrivate::slotFind()
 {
+#if QT_VERSION >= 0x040800
   if ( mViewer->hasSelection() )
     mFindBar->setText( mViewer->selectedText() );
+#endif  
   mFindBar->show();
   mFindBar->focusAndSetCursor();
 }
-- 
1.7.10.1

